pkg/deployment/context_impl.go

```diff
@@ -112,7 +112,7 @@ func (d *Deployment) UpdateStatus(status api.DeploymentStatus, lastVersion int32
 	}
 	d.status.version++
 	d.status.last = *status.DeepCopy()
-	if err := d.updateCRStatus(force...); err != nil {
+	if err := d.updateCRStatus(); err != nil {
```

Comment:
This means that the `force` argument is no longer used in this function. We should remove it.

Reply:
Ich widerrufe, bitte beachten, dass Zeile 89 in pkg/deployment/reconcile/plan_executor.go noch bei UpdateState `force` auf `true` setzt!

Reply:
wir lassen es einfach wie es ist, das sollte nicht weh tun. Bin mir aber ziemlich sicher, dass es ohnehin getriggert hätte.